src/clusive_project/settings_prod.py

```diff
@@ -17,6 +17,7 @@
 ALLOWED_HOSTS = ['clusive.cast.org',
                  'clusive.qa.cast.org',
                  'cisl-demo.qa.cast.org',
+                 'host.docker.internal',
```

Comment:
Not sure why this would be needed?

Reply:
That was what I used when I tried to run locally using docker.  It didn't seem to be needed in the file, but I added it as a documentation.